README.md

```diff
@@ -89,17 +89,255 @@ Name of the application pool.
 
 The state of the ApplicationPool. Must be either `Started` or `Stopped`
 
+#### `auto_start`
+
+When true, indicates to the World Wide Web Publishing Service (W3SVC) that the application pool should be automatically started when it is created or when IIS is started. Valid options `true` or `false`
+
+#### `clr_config_file`
+
+Specifies the .NET configuration file for the application pool
+
+#### `enable32_bit_app_on_win64`
+
+When true, enables a 32-bit application to run on a computer that runs a 64-bit version of Windows. Valid options `true` or `false`
+
+#### `enable_configuration_override`
+
+When true, indicates that delegated settings in Web.config files will processed for applications within this application pool. When false, all settings in Web.config files will be ignored for this application pool. Valid options `true` or `false`
+
 #### `managed_pipeline_mode`
 
-The managedpipelinemode of the ApplicationPool. Must be either `Integrated` or `Classic`
+Specifies the request-processing mode that is used to process requests for managed content. Must be either `Integrated` or `Classic`
```

Comment:
I think Integrated and Classic might actually be string values? If so, they should be in single quotes instead of backticks.